### PR TITLE
issue67: fixing server problem when Mongo is not running

### PR DESCRIPTION
**Pull request by carloscarcamo**
- Replacing [Monk](https://github.com/Automattic/monk) in favor of [Mongoose](https://github.com/Automattic/mongoose) as MongoDB connector, discussion here #68 
- Add new Mongoose model: tweets
- NPM Command for server development using [nodemon](https://github.com/remy/nodemon)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
   "main": "app.js",
   "scripts": {
     "server-start": "node app.js",
+    "server-start-dev": "nodemon --watch app.js --watch server/",
```
Comment (anvk) [inline package.json]: that's an interesting addition. What is it for? I'm just curious.

**Comment by anvk** [comment]
This pull request looks really good. Outstanding job. I only have 1 comment-question there.
Also if you could rebase your 1 commit and change the message to something like `issue67: changing monk to mongoose` so that if needed we can find this change in the history in the future (in case if we will go back to mongoose or just for the history consistency purposes.

File: package.json (hunk 1 above)
Comment (carloscarcamo) [inline package.json]: This is for use during development, is like doing node app.js but:
>nodemon will watch the files in the directory in which nodemon was started, and if any files change, nodemon will automatically restart your node application.

**Comment by carloscarcamo** [comment]
@anvk is about changing the first commit message: 
> Replacing Monk in favor of Mongoose as MongoDB connector

to: `issue67: changing monk to mongoose`?, or the pull request message:
>issue67: fixing server problem when Mongo is not running

to: `issue67: changing monk to mongoose`?

**Comment by anvk** [comment]
Hey I was talking about the commit for this pull request https://github.com/JaredHawkins/TweetGeoViz/pull/69/commits
Take a look the message is `Update`. Once it is changed I will merge your work right away!

File: package.json (hunk 1 above)
Comment (anvk) [inline package.json]: Oh nice! To be honest I did not know about this, good to learn something new.

**Comment by carloscarcamo** [comment]
Ok. I will change it!

**Comment by carloscarcamo** [comment]
done!

**Comment by anvk** [comment]
merged !

**Pull request merged**